File: 백준/Bronze/2309. 일곱 난쟁이/일곱 난쟁이.js
const input = require('fs').readFileSync('/dev/stdin').toString().trim().split("\n").map(Number).sort((a,b)=>a-b);


const initalValue = 0;
const sumAll = input.reduce(
    (acc, cur) => acc + cur, initalValue
);

let arr1 = []
const subtract = sumAll - 100
for (let i = 0; i < 9; i++) {
    let a = subtract-input[i]
    if (input.includes(a)) {
        arr1.push(input[i])
        arr1.push(a)
        break;
    }
}


console.log(input.filter(x => !arr1.includes(x)).join("\n"))

File: 백준/Bronze/2309. 일곱 난쟁이/일곱 난쟁이.py
import sys
input = sys.stdin.readline

lst = []
[lst.append(int(input())) for i in range(9)]
lst_sum = sum(lst)
lst.sort()

for i in range(len(lst)):
    for j in range(i+1, len(lst)):
        if lst_sum - lst[i] - lst[j] == 100:
            for k in range(len(lst)):
                if k == i or k == j:
                    pass
                else:
                    print(lst[k])
            exit()